packages/@aws-cdk/aws-efs/lib/file-system.ts

```diff
@@ -0,0 +1,38 @@
+import * as ec2 from '@aws-cdk/aws-ec2';
+import {IResource} from "@aws-cdk/core";
+
+/**
+ * Interface to implement AWS File Systems.
+ */
+export interface IFileSystem extends IResource, ec2.IConnectable {
```

Comment:
The intention of adding this interface is to have a common interface for all AWS file systems (FSx windows, lustre). This can be moved to `aws-core` once we have construct for another type of file system.

Reply:
updated.